src/app/calendar/_components/CalendarSection.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+import { Calendar } from "@/components/Calendar";
+import { useState } from "react";
+import { CalendarTimeManager } from "./CalendarTimeManager";
+import { CalendarDaySection } from "./CalendarDaySection";
+
+enum ReservationType {
```

Comment:
Enums are evil. Don't use them in typescript
https://www.youtube.com/watch?v=jjMbPt_H3RQ